src/index.js

```diff
@@ -69,6 +69,7 @@ class Game extends React.Component {
       history: history.concat([
         {
           squares: squares,
+          location: i,
```

Comment:
This is a test review. Please address. Thanks!

Reply:
This is great.  I appreciate testing! Thanks.

Reply:
Ok looks good, thanks for taking care of this.